Sources/FoundationEssentials/URL/URLParser.swift

```diff
@@ -158,17 +158,16 @@ package protocol UIDNAHook {
     static func decode(_ host: some StringProtocol) -> String?
 }
 
+dynamic package func _uidnaHook() -> UIDNAHook.Type? {
```

Comment:
Nit:
From the [implementation PR](https://github.com/swiftlang/swift/pull/20333/files#diff-d425fbf14611e9cd6298e6ef29c1c915d9c757e739b9095e7b097858e6ee7740R244) it looks like dynamic replacement would work with properties too. So if we want to keep this as a read-only `var` I suppose we can keep doing that. I'm fine with this though.

Reply:
Ah yeah I could do that too, I'm fine with either - happy to change it if you have a preference for the variable spelling